src/dash/DashMetrics.js

```diff
@@ -345,7 +345,7 @@ function DashMetrics(config) {
      * @instance
      * @ignore
      */
-    function addHttpRequest(request, responseURL, responseStatus, responseHeaders, traces) {
+    function addHttpRequest(request, responseURL, responseStatus, responseHeaders, cmsd, traces) {
```

Comment:
Suggesting to add `cmsd` to the end. In case existing applications overwrite this method they now report `cmsd` instead of `traces`